cloudtests.py

```diff
@@ -62,16 +62,34 @@ def test_net_name(self):
 class TestJournalWritten(unittest.TestCase):
 
     def test_journal_written(self):
-        """Test to check if journal gets written to disk"""
+        """
+        Test to check if journal gets written to disk
+        And make sure that journal logs get written to /var
```

Comment:
Also mention that this test should definitely run on first boot - and also possible after reboot